### ISSUE TITLE
How loginWithToken work?

### ISSUE THREAD

**Issue by safwanidrees** [PurpleSoftSrl/sumup_flutter_plugin#28]
I want to use loginWithToken in the app. How it will work? How I get that token  from sum up the dashboard?

**Comment by sstasi95** [comment]
Hi,

I was able to obtain a valid access token with [this method](https://github.com/sumup/sumup-android-sdk/issues/45#issuecomment-586647889), i.e. using grant_type password and providing username and password.

Hope it helps,

Simone

**Comment by safwanidrees** [comment]
> 

Thanks, I got a token from that 
But that token gives an error the invalid token in loginWithToken Function.

**Comment by sstasi95** [comment]
I've just tried again today and I can confirm that it works for me, are you sure the token was still valid? It should expire after a hour. If it's not because of the token expiration, honestly I don't know what can be the problem.

**Comment by rivella50** [comment]
@sstasi95 Do i see this correctly, that `loginWithToken` is meant for the merchant, but not for an employee of the SumUp account?
I.e. an employee can only be correctly handled if he is sent to `Sumup.login()` and enters there his client username and password?

**Comment by sstasi95** [comment]
@rivella50  Yes I think you're right. From SumUp documentation it's not clear, but I couldn't find a way to loginWithToken with an employee.

**Comment by sebyspi** [comment]
Hi have the same issue. I contacted the support but they redirected me here. Is there any solution?
